kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -1778,9 +1778,9 @@ Precompiled Contracts
          <callData> DATA </callData>
          <output> _ => #ecrec(#range(DATA, 0, 32), #range(DATA, 32, 32), #range(DATA, 64, 32), #range(DATA, 96, 32)) </output>
 
-    syntax Bytes ::= #ecrec ( Bytes , Bytes , Bytes , Bytes ) [function, smtlib(ecrec)]
-                   | #ecrec ( Account )                       [function]
- // --------------------------------------------------------------------
+    syntax Bytes ::= #ecrec ( Bytes , Bytes , Bytes , Bytes ) [function, smtlib(ecrec), total]
+                   | #ecrec ( Account )                       [function, total]
+ // ----------------------------------------------------------------------------
```

Comment:
If `#sender` is not total, I believe we cannot mark this as total either.

Reply:
I am trying to substantiate in the PR description that `#sender` with the `Bytes x Bytes x Bytes x Bytes -> Bytes` signature is indeed total.